src/providers/monsterapi/api.ts

```diff
@@ -0,0 +1,18 @@
+import { ProviderAPIConfig } from '../types';
+
+const MonsterAPIApiConfig: ProviderAPIConfig = {
+  getBaseURL: () => 'https://llm.monsterapi.ai/v1',
+  headers: ({ providerOptions }) => {
+    return { Authorization: `Bearer ${providerOptions.apiKey}` };
+  },
+  getEndpoint: ({ fn }) => {
+    switch (fn) {
+      case 'generate':
```

Comment:
This case should also use chatComplete instead of generate. And the default should return an empty string. Basically this:
```
switch (fn) {
      case 'chatComplete':
        return '/generate';
      default:
        return '';
    }
```

Reply:
Done